### DISCUSSION TITLE
Inquiry Regarding GitHub Campus Expert Programme Application Status

### DISCUSSION THREAD

**Discussion by codiearyan** [community/community#113745] [GitHub Education] [answered]
### Hi everyone,

I submitted my application on 12th April, 2024 and have not yet received any updates regarding its status. I got a mail on 12th March saying this year there a lot of applicants, So my application will be reviewed till 19th March. I am eagerly waiting..

**Comment by samus-aran** [comment, marked as answer]
Hi there 👋

We’re sorry about the mail not reaching you. There are recent changes to mass email rules that may be affecting the delivery of some messages. However, all applicants can always check their status by logging in at https://education.github.com/campus_experts.
